datahub-frontend/app/controllers/api/v2/CorpUser.java

```diff
@@ -69,7 +69,7 @@ public Result updateCorpUserEditableInfo(@Nonnull String corpUserUrn) {
         } catch (URISyntaxException e) {
             return unauthorized("Invalid urn");
         }
-        if (!corpUser.getUsernameEntity().equals(username)) {
+        if (!corpUser.getUsername().equals(username)) {
```

Comment:
Internally the urn utils deprecates `getUsername()` and recommends using `getUsernameEntity()`

Reply:
Yeah, I did undo it, then i pushed the wrong branch. Oops.